StandaloneFMU/Makefile

```diff
@@ -18,6 +18,13 @@ XXSIM_SOURCES = EulerAngles.c \
 	xxmatrix.c \
 	xxmodel.c \
 	xxsubmod.c \
+%IF%%EQ(INTEGRATION_METHOD_NAME,VodeAdams)%
```

Comment:
Note that the %EQ% construction is only available since 20-sim 4.6
This means that you have update the minimal required 20-sim version too in the Targets.ini